### PR TITLE
:heavy_plus_sign: Add django-markup and markdown to requirements

### PR DESCRIPTION
**Pull request by stevenbal**
Required for: https://github.com/maykinmedia/objects-api/pull/222#discussion_r669522516

The packages `django-markup` and `markdown` are required to generate the notifications page, but were not included in `setup.cfg`

### CODE REVIEW

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
I'd rather see those in the `extras` as an optional dependency. If you don't use the views/templates or have overridden versions, then the dependencies are just dead weight.

And extras group `markdown-docs` would make most sense, so we can then essentially do `pip install vng-api-common[markdown-docs]`.

File: setup.cfg [hunk 1]
```diff
@@ -47,7 +47,7 @@ install_requires =
     iso-639
     isodate
     oyaml
-    PyJWT
+    PyJWT~=1.7
```
Comment (stevenbal) [inline setup.cfg:50]: CI seemed to be installing PyJWT==2.1.0

Comment (sergei-maertens) [reply setup.cfg:50]: Not okay to pin this, it's breaking existing installations. We should deal with the changes, I've already done that in gemma-zds-client as well so you can look at that for inspiration.

File: vng_api_common/tests/auth.py [hunk 2]
```diff
@@ -59,7 +59,6 @@ def generate_jwt_auth(
         "user_representation": user_representation,
     }
     encoded = jwt.encode(payload, secret, algorithm="HS256")
-    encoded = encoded.decode("ascii")
```
Comment (stevenbal) [inline vng_api_common/tests/auth.py:62]: @sergei-maertens Is it okay to assume that these tests will be run using PyJWT >= 2.0.0, or should it still be compatible with PyJWT<2.0.0?

Comment (sergei-maertens) [reply vng_api_common/tests/auth.py:62]: you can assume tests run with old PyJWT version, but the existing projects may have PyJWT < 2.0 so the old variant should still work :-)

**Comment by stevenbal** [comment]
@sergei-maertens is there anything else that has to happen for this PR?

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**